plugins/azure/sqlserver/enableSendEmailToAdmins.js

```diff
@@ -0,0 +1,66 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Send Emails to Admin and Owners Enabled',
+    category: 'SQL Server',
+    domain: 'Databases',
+    description: 'Ensures that Send Emails to admins and owners is enabled for SQL Servers',
+    more_info: 'Vulnerability Assessment (VA) scan reports and alerts will be sent to email ids configured at \'Send scan reports to\'. This may help in reducing time required for identifying risks and taking corrective measures.',
```

Comment:
```suggestion
    more_info: 'Vulnerability Assessment (VA) scan reports and alerts will be sent to email addresses configured in the "Send scan reports to" section. This may help in reducing time required for identifying risks and taking corrective measures.',
```